explore/src/main/scala/explore/observationtree/ObsList.scala

```diff
@@ -267,11 +283,14 @@ object ObsList:
                   clazz = ExploreStyles.ButtonSummary
                 )
               ),
-              Tree(
-                treeNodes,
-                renderItem,
-                expandedKeys = expandedGroups.get,
-                onToggle = expandedGroups.set
+              <.div(
+                ^.overflow := "auto",
```

Comment:
Why do we need the overflow?

Reply:
To actually make the list scrollable. Before it came from a parent, but the wrapping in the div is needed to prevent scroll-resetting. I spent a week on this, don't ask me why 🪄

Reply:
I can certainly believe that